jscomp/ext/ext_path.ml

```diff
@@ -322,3 +322,32 @@ let find_package_json_dir cwd  =
   find_root_filename ~cwd  Literals.bsconfig_json
 
 let package_dir = lazy (find_package_json_dir (Lazy.force cwd))
+
+let getchdir s =
+  let p = Sys.getcwd () in
+  Unix.chdir s;
+  p
+
+let normalize s =
+  try getchdir (getchdir s) with _ -> s
```

Comment:
This assume `s` exists?

Reply:
I've removed try/catch. This resolves symlink to dir to actual path